docker-compose-proxy.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.3'
```

Comment:
I'm not sure what we're talking about here.

the goal of a single `docker-compose up` is a local-dev-only goal. CI/CD will do all the extra things we'd need for testing and production deploy.

`docker-compose` cli is for dev and testing only. It doesn't understand multi-server clusters like Swarm

`docker-compose.yml` file is for dev, test, and prod docker Swarm. (using `docker stack deploy` in swarm)

We'll likely use override files with `docker-compose config` for production deploy, but not sure yet until we get everything needed into compose file and see what the dev-vs-prod differences are. We can't use extends, but [overrides work great](https://docs.docker.com/compose/extends/#multiple-compose-files).

Does that help?